src/store/getters/resource-getters.ts

```diff
@@ -27,6 +27,11 @@ export const getPeopleAndRolesResource = (state: StateIF): any => {
   return state.resourceModel.peopleAndRoles
 }
 
+/** The Incorportaion Articles */
```

Comment:
Small typo.

Reply:
Hehe, now you have another typo.

Reply:
Oh my gosh this is getting embarassing 🤕

Reply:
Third time's a charm :)